README.md

```diff
@@ -98,71 +98,70 @@ Default variables values are as follows:
 
 ### Deployment
 - To deploy resources run `setup` command and specify cloud name (aws|gcp|all):
-
-     E.g: `./mcadm.sh setup aws`
+  ```bash
+  ./mcadm.sh setup aws
+  ```
 
 ### Cleanup
 - To remove all the resources deployed run the `destroy` command:
-
-      E.g: `./mcadm.sh destroy manager`
+  ```bash
+  ./mcadm.sh destroy manager
+  ```
 
 ### Adding nodes to a cluster
-- To add a node (etcd|control|worker) to an existing cluster run the add command:
-
-      E.g: add a new worker node to an AWS cluster:
-      `./mcadm.sh add wnode config/test/test-aws-cluster.yaml`
+- To add a node (etcd | control | worker) to an existing cluster run the add command:
+  ```bash
+  ./mcadm.sh add wnode config/test/test-aws-cluster.yaml # add new worker node to an AWS cluster
+  ```
 
 ### Removing nodes from an existing cluster
 - To remove a node (etcd|control|worker) from a cluster run the destroy command:
-
-      E.g: remove a node from an AWS cluster:
-      `./mcadm.sh destroy node config/test/test-aws-cluster.yaml`
-
-     A prompt will ask you which node to remove 
+  ```bash
+  ./mcadm.sh destroy node config/test/test-aws-cluster.yaml # remove a node from an AWS cluster
+  ```
+  A prompt will ask you which node to remove
 
 ### Getting information about manager or cluster
 - To get information about a manager or a cluster run the get command:
+  ```bash
+  ./mcadm.sh get manager # information about the cluster manager
+  ```
 
-      E.g: to get information about the cluster manager:
-      `./mcadm.sh get manager`
-
-### help
+### Help
 - Run `./mcadm.sh help` to see details about script usage
-- The information about the Kubernetes Cluster Manager will be shown in the console output. Rancher manager UI user:
-  admin
 
-      E.g:
-      `rancher_access_key = token-1abcd`
-      `rancher_secret_key = xyz1xyz2xyz`
-      `rancher_url = https://3.4.1.2`
+## Tips and tricks
 
-- Use SECURE_SOURCE_IP to add any IP to the monitoring services firewalls so Zabbix Web and Kibana UI can be reached
+- The information about the Kubernetes Cluster Manager will be shown in the console output. Rancher manager UI user:
+  `admin`
+  ```
+  rancher_access_key = token-1abcd
+  rancher_secret_key = xyz1xyz2xyz
+  rancher_url = https://3.4.1.2
+  ```
+- Use `SECURE_SOURCE_IP` to add any IP to the monitoring services firewalls so Zabbix Web and Kibana UI can be reached
   from that IP
-- The Zabbix Web Frontend can be accessed on a browser using the public Zabbix Server IP. User: Admin
-- Kibana UI can be accessed on a browser using the public ELK Server IP and port 5601
+- The Zabbix Web Frontend can be accessed on a browser using the public Zabbix Server IP. User: `Admin`
+- Kibana UI can be accessed from a browser using the public ELK Server IP and port 5601
 - A multi-cloud setup on both AWS and GCP currently only allows the creation of Zabbix and Elastic Stack servers on AWS.
```

Comment:
I was uncertain about the intent here. Checking the what the help command does (which is the section where these previously lived) does not do or relate to any of the above points, so suggesting this 'miscellaneous' section instead